content/en/tracing/trace_collection/library_config/rust.md

```diff
@@ -0,0 +1,79 @@
+---
+title: Configuring the Rust Tracing Library
+code_lang: rust
+type: multi-code-lang
+code_lang_weight: 80
+further_reading:
+    - link: 'https://github.com/DataDog/dd-trace-rs'
+      tag: "Source Code"
+      text: 'Source code'
+    - link: "/tracing/trace_collection/trace_context_propagation/"
+      tag: "Documentation"
+      text: "Propagating trace context"
+    - link: 'tracing/glossary/'
+      tag: 'Documentation'
+      text: 'Explore your services, resources and traces'
+---
+
+After you [set up the Rust SDK][1] with your application, you can optionally configure it using the following environment variables.
+
+## Unified Service Tagging
+
+It is recommended to use `DD_ENV`, `DD_SERVICE`, and `DD_VERSION` to set `env`, `service`, and `version` for your services.
```

Comment:
The full list of the configuration options and default values are now documented here https://github.com/DataDog/dd-trace-rs/blob/main/supported-configurations.json